TwitterTest.py

```diff
@@ -0,0 +1,53 @@
+"""
+MP3: Chef Gordon Ramsay's Sentiment Analysis on Twitter
+@ilya-besancon
+February 2017
+"""
+import twitter
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+
+def main():
+    """ Gets last 200 tweets from Chef's Twitter Page """
+    CONSUMER_KEY = '3sqEK5AwQUI32isQAV4LGTdDa'
+    CONSUMER_SECRET = '9xjKjy5oovPD5fBeH3JwI4gBjPVwSmaqFUAXW0eLEE7A0yvTwZ'
+    ACCESS_TOKEN_KEY = '833009544758444033-K7Kb08YAz5uMHXc2jFzCDeiCq5SJhyQ'
+    ACCESS_TOKEN_SECRET = 'gPqGBcsY0YDCVY3wcfVIj2DhgV6y3JtRVMRxDQjpr8yNE'
```

Comment:
Not a good idea to include API credentials like this in your file. This is a private information that should not be put in a public place like GitHub. Take a look at an in-class note about this.